CoordgenFragmentBuilder.cpp

```diff
@@ -522,15 +531,7 @@ void CoordgenFragmentBuilder::buildRing(sketcherMinimizerRing* ring) const
             for (unsigned int i = 0; i < atoms.size(); i++) {
                 atoms[i]->setCoordinates((*targetCoords)[i]);
             }
-            // TODO forbid if double bond to ring
-            if (ring->fusedWith.size() == 1 &&
-                (*fusionAtoms.begin())->hasNoStereoActiveBonds() &&
-                (*fusionAtoms.rbegin())->hasNoStereoActiveBonds()) {
-                auto* dof = new CoordgenFlipRingDOF(ring, fusionAtoms);
```

Comment:
Do you think removing the `CoordgenFlipRingDOF` will have a negative effect on some other molecules? What is the purpose of that DOF?

Reply:
It should be safe. It is intended to swap between structures like the ones here below, in case for instance there were substituents on the nitrogen, but the way it was implemented I don't think it would ever work.
<img width="340" alt="Screenshot 2021-06-10 at 10 18 20" src="https://user-images.githubusercontent.com/10059176/121490593-76022080-c9d5-11eb-9acc-56247ff57654.png">